README.md

```diff
@@ -1 +1,46 @@
-##Network Service
+# Network Service 
+
+Network layer for creating different set of network requests like GET, POST, PUT, DELETE 
+⚠️ The network layer is under active development.
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) | | 
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) | |
+| ![Swift](https://img.shields.io/badge/Swift-5.8-orange) | |
+| ![SDKs for iOS](https://img.shields.io/badge/SDKs%20for%20iOS-15%2B-lightgrey) | |
+| ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-yellow) | [Link](https://github.com/nicklockwood/SwiftFormat) |
+| ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)| [Link](https://github.com/realm/SwiftLint) |
```

Comment:
Do we need these lines (regarding linters) here?

Reply:
no
fixed

Reply:
What was fixed then?
I still see these lines in the table.

Reply:
sorry misunderstood, removed linters because they are not used in SPM package